test-app/tests/integration/components/bs-dropdown/menu-test.js

```diff
@@ -26,10 +31,28 @@ module('Integration | Component | bs-dropdown/menu', function (hooks) {
     assert.dom('.dropdown-menu').exists('menu has dropdown-menu class');
     assert
       .dom('.dropdown-menu')
-      .hasClass('dropdown-menu-right', 'menu has dropdown-menu-right class');
+      .hasClass(alignmentClass, `menu has ${alignmentClass} class`);
     assert.dom('.dropdown-menu').hasText('Something');
   });
 
+  test('dropdown has correct class for @align="left"', async function (assert) {
+    await render(
+      hbs`<BsDropdown as |dd|><dd.menu
+    @align='left'
+    @isOpen={{true}}
+    @toggleElement={{this.element}}
+  >Something</dd.menu></BsDropdown>`,
+    );
+
+    let alignmentClass = versionDependent(
+      'dropdown-menu-left',
+      'dropdown-menu-start',
+    );
+    assert
+      .dom('.dropdown-menu')
+      .hasClass(alignmentClass, `menu has ${alignmentClass} class`);
```

Comment:
I think we do t expect it to have a class in BS4 and 5.

- https://getbootstrap.com/docs/5.3/components/dropdowns/#menu-alignment
- https://getbootstrap.com/docs/4.6/components/dropdowns/#menu-alignment

I think we should assert that it does not have any class except `.dropdown-menu`. Not sure how well that's supported by qunit-dom.

Reply:
QUnit DOM may not, but I'll do the assertions on the classlist.